conans/client/rest/rest_client.py

```diff
@@ -171,7 +182,7 @@ def get_package(self, package_reference, dest_folder):
         file_paths = self.download_files_to_folder(urls, dest_folder, self._output)
         return file_paths
 
-    def upload_conan(self, conan_reference, the_files, retry, retry_wait):
+    def upload_conan(self, conan_reference, the_files, retry, retry_wait, skip_src_tgz):
```

Comment:
Same here, a skip parameter. Remote manager will pass in the parameter list the tgz name